addons/hr_recruitment_skills/__manifest__.py

```diff
@@ -12,6 +12,7 @@
         'security/hr_recruitment_skills_security.xml',
         'views/hr_applicant_views.xml',
         'views/hr_applicant_skill_views.xml',
```

Comment:
Do we need another file `'views/hr_recruitment_skills_views.xml' ` for just one menu item? :thinking: 

IMO, we could as well put it here. Unless you strongly disagree, let's do so ^^

Reply:
I mildly disagree but ok let's do it like that